src/app/components/elements/MicrosoftSignInButton.tsx

```diff
@@ -0,0 +1,17 @@
+import React from "react";
+import {Button} from "reactstrap";
+import {handleProviderLoginRedirect, useAppDispatch} from "../../state";
+import {siteSpecific} from "../../services";
+
+// Button prompting the user to sign in via Microsoft
+export const MicrosoftSignInButton = () => {
+    const dispatch = useAppDispatch();
+
+    const logInWithMicrosoft = () => {
+        dispatch(handleProviderLoginRedirect("MICROSOFT"));
+    };
+
+    return <Button className={"position-relative"} block outline color={siteSpecific("primary", undefined)} style={{borderColor: "#545461", borderWidth: "1.5px"}} onClick={logInWithMicrosoft}>
```

Comment:
```suggestion
    return <Button color="keyline" onClick={logInWithMicrosoft}>
```
This is the Fluent-style approach to outlined buttons we have on the redesign. Looks identical!

Reply:
for now I'll just go ahead with inline styles. for later, have you got any suggestions for cleaning this up?